### PR TITLE
WIP: [OCPBUGS-29968] - Generated ZTP manifests have wrong/missing Group/Version/Kind

### PR DESCRIPTION
**Pull request by bmanzari**

### CODE REVIEW

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetal-sno-agent-ipv4-static-connected-fips-f7

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetal-pxe-sno-agent-ipv4-static-connected-f14

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetal-pxe-sno-agent-ipv4-static-connected-f14

**Comment by bmanzari** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-nightly-baremetal-pxe-sno-agent-ipv4-static-connected-f14

File: ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh [hunk 1]
```diff
@@ -160,17 +160,24 @@ grep -v "password\|username\|pullSecret" "${SHARED_DIR}/agent-config.yaml" > "${
...
-#echo "Creating manifests..."
-#oinst agent create cluster-manifests
+echo "Creating manifests..."
+oinst agent create cluster-manifests
+
+mkdir -p "${ARTIFACT_DIR}/cluster-manifests/"
+mkdir -p "${INSTALL_DIR}/openshift/"
+
+cp -rT "${INSTALL_DIR}/cluster-manifests/" "${ARTIFACT_DIR}/cluster-manifests/"
+cp -rT "${INSTALL_DIR}/cluster-manifests/" "${INSTALL_DIR}/openshift/"
```
Comment (zaneb) [inline ci-operator/step-registry/baremetal/lab/agent/install/baremetal-lab-agent-install-commands.sh:170]: You don't want to copy anything to the `openshift` directory.

**Pull request closed without merging**